### PR TITLE
Fix quicktests

### PR DESCRIPTION
**Pull request by crmorford**
I made it so that all the tests are identical between branches (sometimes they would show different random data) and I got rid of all the XYDragbox warnings.

### CODE REVIEW

File: quicktests/overlaying/tests/functional/formatter.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 function makeData() {
   "use strict";
 
-  return [makeRandomData(50), makeRandomData(50)];
+  return [makeRandomData(5), makeRandomData(50)];
```
Comment (bluong) [inline quicktests/overlaying/tests/functional/formatter.js]: Why two different numbers here?

File: quicktests/overlaying/tests/functional/formatter.js [hunk 2]
```diff
@@ -12,7 +12,17 @@ function run(svg, data, Plottable) {
     };
 
 
-  var big_numbers = data[0].slice(0, 5);
+
+  var deep_copy = function(from, to){
```
Comment (bluong) [inline quicktests/overlaying/tests/functional/formatter.js]: Probably want to rename to deepCopyPoints and then wrap to a util like makeRandomData?

File: quicktests/overlaying/tests/functional/formatter.js [hunk 3]
```diff
@@ -12,7 +12,17 @@ function run(svg, data, Plottable) {
     };
 
 
-  var big_numbers = data[0].slice(0, 5);
```
Comment (bluong) [inline quicktests/overlaying/tests/functional/formatter.js]: Why can't we use this?

File: quicktests/overlaying/tests/interactions/drag_boxes.js [hunk 4]
```diff
@@ -34,7 +34,7 @@ function run(svg, data, Plottable) {
   var table = new Plottable.Component.Table([[plot1, plot2, plot3]]);
 
   table.renderTo(svg);
-  makeVerifyingDragBox(new Plottable.Interaction.XYDragBox(), plot1);
-  makeVerifyingDragBox(new Plottable.Interaction.XDragBox() , plot2);
-  makeVerifyingDragBox(new Plottable.Interaction.YDragBox() , plot3);
+  makeVerifyingDragBox(new Plottable.Interaction.DragBox(), plot1);
+  makeVerifyingDragBox(new Plottable.Interaction.DragBox() , plot2);
+  makeVerifyingDragBox(new Plottable.Interaction.DragBox() , plot3);
```
Comment (bluong) [inline quicktests/overlaying/tests/interactions/drag_boxes.js]: Reason for this change?

File: quicktests/overlaying/tests/basic/basic_area.js [hunk 5]
```diff
@@ -12,10 +12,19 @@ function run(svg, data, Plottable) {
...
   function getY(d) { return d.y; }
 
-  var dataseries = data[0].slice(0, 20);
-  var dataseries_top = data[1].slice(0, 20);
+  var deep_copy = function(from, to){
+    var deep_copy_xy = function(d){
+      to.push({'x': d.x, 'y': d.y});
+    };
+    from.forEach(deep_copy_xy);
+  };
```
Comment (bluong) [inline quicktests/overlaying/tests/basic/basic_area.js]: Maybe wrap to util

File: quicktests/overlaying/tests/basic/basic_area.js [hunk 6]
```diff
@@ -12,10 +12,19 @@ function run(svg, data, Plottable) {
...
 
-  for (var i = 0; i < 20; ++i) {
+  var dataseries = [];
+  deep_copy(data[0], dataseries);
+  var dataseries_top = [];
+  deep_copy(data[1], dataseries_top);
```
Comment (bluong) [inline quicktests/overlaying/tests/basic/basic_area.js]: Why is this necessary?

File: quicktests/overlaying/tests/functional/formatter.js (hunk 1 above)
Comment (crmorford) [inline quicktests/overlaying/tests/functional/formatter.js]: I only used the first one. If I change both to 5, does that make it correct?

File: quicktests/overlaying/tests/functional/formatter.js (hunk 3 above)
Comment (crmorford) [inline quicktests/overlaying/tests/functional/formatter.js]: This file was doing a shallow copy, and then manipulating data. Because the data was manipulated twice, the two quicktests looked different from each other, which made them difficult to compare. I deleted .slice() from a lot of files in favor of deep copies.

File: quicktests/overlaying/tests/interactions/drag_boxes.js (hunk 4 above)
Comment (crmorford) [inline quicktests/overlaying/tests/interactions/drag_boxes.js]: Errors in the console. "Use DragBox instead, XYDragbox has been deprecated"

File: quicktests/overlaying/tests/basic/basic_area.js (hunk 6 above)
Comment (crmorford) [inline quicktests/overlaying/tests/basic/basic_area.js]: Same as formatters.js, manipulations on the data require deep copies to be made instead of shallow copies, so that the two examples are identical.

File: quicktests/overlaying/tests/functional/formatter.js (hunk 2 above)
Comment (crmorford) [inline quicktests/overlaying/tests/functional/formatter.js]: What does "wrap to a util" mean?

File: quicktests/overlaying/tests/functional/formatter.js (hunk 1 above)
Comment (bluong) [inline quicktests/overlaying/tests/functional/formatter.js]: Let's just be consistent jic

File: quicktests/overlaying/tests/functional/formatter.js (hunk 2 above)
Comment (bluong) [inline quicktests/overlaying/tests/functional/formatter.js]: write this function into the same place as makeRandomData is written so that you don't have to rewrite this function every time.

File: quicktests/overlaying/tests/functional/formatter.js (hunk 3 above)
Comment (bluong) [inline quicktests/overlaying/tests/functional/formatter.js]: Got it

**Comment by bluong** [comment]
Ok now I understand the need for a deep copy.  Just make the changes above.

File: quicktests/exampleUtil.js [hunk 7]
```diff
@@ -109,3 +109,10 @@ function generateHeightWeightData(n) {
 
   return data;
 }
+
+var deep_copy = function(from, to){
```
Comment (bluong) [inline quicktests/exampleUtil.js]: Can we rename this to deep_copy_points

File: quicktests/exampleUtil.js [hunk 8]
```diff
@@ -109,3 +109,10 @@ function generateHeightWeightData(n) {
...
+  var deep_copy_xy = function(d){
+    to.push({'x': d.x, 'y': d.y});
+  };
+  from.forEach(deep_copy_xy);
```
Comment (bluong) [inline quicktests/exampleUtil.js]: For cleanliness I don't think we need to extract the function out and we can inline it instead.  No big deal though

**Comment by bluong** [comment]
Minor at this point

**Comment by crmorford** [comment]
can i merge? (after release)
